### DISCUSSION TITLE
Release: v3.0.0-dev-preview.9

### DISCUSSION THREAD

**Discussion by rexxars** [sanity-io/sanity#3364] [Announcements]
# 🐛 Notable bugfixes
- Fixes issue where schema errors would not be displayed in error boundary
- Fixes reference inputs not showing create buttons in certain contexts
- Fixes a few unstyled UI elements using system default font
- Fixes change indicators not appearing in all contexts
- Fixes object input not respecting `columns` option

# 📓 Full changelog
Author | Message | Commit
------------ | ------------- | -------------
Per-Kristian Nordnes | refactor(form/inputs): add missing change indicators | 0dbfc338a
Per-Kristian Nordnes | fix(desk,form): restore default draft change indicators | bfcfb3b8b
Herman Wikner | feat(presence): fix tooltip scheme in DocumentPreviewPresence | e60850e18
Herman Wikner | feat(studio): close NavDrawer when changing workspace, add tooltip to WorkspaceMenuButton | b02476aba
Marius Lundgård | test(sanity): fix path to config factory | f64f2dbb9
Espen Hovlandsdal | chore(deps): get rid of remaining prop-types usage | 99019a9fb
Espen Hovlandsdal | chore(deps): replace pluralize with pluralize-esm | 2e0cd7ade
Espen Hovlandsdal | chore(deps): replace get-random-values with get-random-values-esm | 0384ae77c
Per-Kristian Nordnes | fix(form/inputs): pt-input: fix opened member finder | 1b2041e38
Herman Wikner | test: add object with columns option to `test-studio` | dba35810a
Herman Wikner | feat(form): add columns support to `ObjectInput` | ab0c4c4d4
Herman Wikner | feat(form): add columns support to `MemberFieldset` | a9ab9379f
Saiichi Shayan Hashimoto | feat(types): add `calendarTodayLabel` to date/datetime schema options (#3355) | 7736b4002
Saiichi Shayan Hashimoto | feat(types): add missing `name` to `SortOrdering` type (#3353) | f9ba13079
Bjørge Næss | chore: upgrade react-rx | fc9cbf13a
Espen Hovlandsdal | chore(deps): normalize versions | 490e30267
Espen Hovlandsdal | chore(components): drop unused version checker component | edfe18a8f
Espen Hovlandsdal | feat(cli): migrate ecommerce template | f2cd91012
Espen Hovlandsdal | chore(examples): migrate ecommerce example studio | 26814afae
Espen Hovlandsdal | chore(examples): drop static folder from clean-studio | 1e756d7fa
Espen Hovlandsdal | chore: add `clean:deps` script to remove all `node_modules` dirs | 164997f6f
Espen Hovlandsdal | fix(desk): add name to default sort orderings | 210c7b677
Espen Hovlandsdal | fix(form): fix typings for `columns` property | 5078c08bf
Espen Hovlandsdal | chore(types): drop part system typings | a20c9fe2a
Espen Hovlandsdal | build(vercel): fix build command for vercel deployments in v3 | 088b9af99
Bjørge Næss | chore(form): rename SanityReferenceInput => StudioReferenceInput | 132d945bd
Bjørge Næss | fix(desk): pass global templates as initial value templates to reference inputs | 568f7dfce
Espen Hovlandsdal | fix(cli): drop outdated install command | ee7ef7e71
Espen Hovlandsdal | feat(cli): throw error on v2 plugin install with no `@sanity/core` installed | 83201a5de
Espen Hovlandsdal | chore(lint): set react version to 17 for eslint-plugin-react | 50aa894fd
Espen Hovlandsdal | chore: update lockfile | 64e91f427
Espen Hovlandsdal | chore: drop unnecessary eslint rule overrides | 455129ad6
Espen Hovlandsdal | chore: disable tsdoc syntax warning for jest config | c9da5d8d0
Espen Hovlandsdal | feat(pkg-utils): allow configuring target per export | bad4a0777
Espen Hovlandsdal | chore: set node target for cli/internal export | cc746cec4
Espen Hovlandsdal | fix: catch workspace resolution errors with error boundary | c5c1ba160
Espen Hovlandsdal | refactor(schema): pad list of all possible types on schema error | d6330a492
Espen Hovlandsdal | feat(screens): preserve newlines in error messages | 06e996101
Espen Hovlandsdal | chore(deps): drop unused semver-compare dependency | 0a80b9c80
Espen Hovlandsdal | test: temporarily silence `flushSync` errors | f8369a8e4
Espen Hovlandsdal | chore(deps): add types package for semver-compare | 2a9fc550d
Herman Wikner | fix(sanity): update presence tooltip ui | af09edd69

<hr /><em>This discussion was created from the release <a href='https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.9'>v3.0.0-dev-preview.9</a>.</em>

**Comment by dorelljames** [comment]
Thanks for the release! 🔥

Testing this out now, hoping the `generateHelpUrl` error has been fixed! Ref: https://github.com/sanity-io/sanity/discussions/3334#discussioncomment-2982977

**Reply by dorelljames** [reply]
And it worked! Awesome. thank you
